gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/common/AbstractTransportServiceStubClassComposer.java

```diff
@@ -1361,7 +1361,7 @@ private void createRequestParamsExtractorBodyForRoutingHeaders(
     for (int i = 0; i < routingHeaderParams.size(); i++) {
       RoutingHeaderRule.RoutingHeaderParam routingHeaderParam = routingHeaderParams.get(i);
       MethodInvocationExpr requestFieldGetterExpr =
-          createRequestFieldGetterExpr(requestVarExpr, routingHeaderParam.fieldName());
+          createRequestFieldGetterExpr(requestVarExpr, routingHeaderParam.fieldName(),false);
```

Comment:
Why we want to pass `false` here? I believe this line is for explicit routing header, which we should cover as well.